tests/pyfunc/test_model_export_with_class_and_artifacts.py

```diff
@@ -150,6 +150,20 @@ def test_predict(sk_model, model_input):
     )
 
 
+@pytest.mark.large
+def test_model_save_behavior_with_preexisting_folders(sklearn_knn_model, tmpdir):
+    sklearn_model_path = os.path.join(str(tmpdir), "sklearn_model_empty_exists")
+    os.makedirs(sklearn_model_path)
+    mlflow.sklearn.save_model(sk_model=sklearn_knn_model, path=sklearn_model_path)
+
+    sklearn_model_path = os.path.join(str(tmpdir), "sklearn_model_filled_exists")
+    os.makedirs(sklearn_model_path)
+    with open(os.path.join(sklearn_model_path, "foo.txt"), "wt") as f:
+        f.write("dummy content")
+    with pytest.raises(MlflowException, match="already exists and is not empty"):
+        mlflow.sklearn.save_model(sk_model=sklearn_knn_model, path=sklearn_model_path)
```

Comment:
Can we move this test in `tests/sklearn/test_sklearn_model_export`?

Reply:
done!